recipe/meta.yaml

```diff
@@ -16,53 +16,65 @@ outputs:
     requirements:
       run:
       - libgcc-devel_linux-64 7.5.*
-      - gcc_linux-64 7.5.*
-      - gcc_impl_linux-64 7.5.*
+      {% for cmp in ["gcc", "gxx", "gfortran"] %}
+      - {{ cmp }}_linux-64 7.5.*
+      - {{ cmp }}_impl_linux-64 7.5.*
+      {% endfor %}
 
   - name: _current_repodata_hack_gcc_linux_64_84
     build:
       skip: True  # [not (linux and x86_64)]
     requirements:
       run:
       - libgcc-devel_linux-64 8.4.*
-      - gcc_linux-64 8.4.*
-      - gcc_impl_linux-64 8.4.*
+      {% for cmp in ["gcc", "gxx", "gfortran"] %}
+      - {{ cmp }}_linux-64 8.4.*
+      - {{ cmp }}_impl_linux-64 8.4.*
+      {% endfor %}
   
   - name: _current_repodata_hack_gcc_linux_aarch64_75
     build:
       skip: True  # [not (linux and (x86_64 or aarch64))]
     requirements:
       run:
       - libgcc-devel_linux-aarch64 7.5.*
-      - gcc_linux-aarch64 7.5.*
-      - gcc_impl_linux-aarch64 7.5.*
+      {% for cmp in ["gcc", "gxx", "gfortran"] %}
+      - {{ cmp }}_linux-aarch64 7.5.*
+      - {{ cmp }}_impl_linux-aarch64 7.5.*
+      {% endfor %}
   
   - name: _current_repodata_hack_gcc_linux_aarch64_84
     build:
       skip: True  # [not (linux and (x86_64 or aarch64))]
     requirements:
       run:
       - libgcc-devel_linux-aarch64 8.4.*
-      - gcc_linux-aarch64 8.4.*
-      - gcc_impl_linux-aarch64 8.4.*
+      {% for cmp in ["gcc", "gxx", "gfortran"] %}
+      - {{ cmp }}_linux-aarch64 8.4.*
+      - {{ cmp }}_impl_linux-aarch64 8.4.*
+      {% endfor %}
   
   - name: _current_repodata_hack_gcc_linux_ppc64le_75
     build:
       skip: True  # [not (linux and (x86_64 or ppc64le))]
     requirements:
       run:
       - libgcc-devel_linux-ppc64le 7.5.*
-      - gcc_linux-ppc64le 7.5.*
-      - gcc_impl_linux-ppc64le 7.5.*
+      {% for cmp in ["gcc", "gxx", "gfortran"] %}
+      - {{ cmp }}_linux-ppc64le 7.5.*
+      - {{ cmp }}_impl_linux-ppc64le 7.5.*
+      {% endfor %}
   
   - name: _current_repodata_hack_gcc_linux_ppc64le_84
     build:
       skip: True  # [not (linux and (x86_64 or ppc64le))]
     requirements:
       run:
       - libgcc-devel_linux-ppc64le 8.4.*
-      - gcc_linux-ppc64le 8.4.*
-      - gcc_impl_linux-ppc64le 8.4.*
+      {% for cmp in ["gcc", "gxx", "gfortran"] %}
```

Comment:
```suggestion
      {% for cmp in ["gcc", "gxx", "gfortran", "libgcc-devel", "libstdxx-devel"] %}
```

Reply:
Right, we can do that for the others too